manuscript/4.0authentication.md

```diff
@@ -55,174 +55,174 @@ While generating a new session, we need to check if a sessions is already active
 Till now we never used any third party library or a framework in this book, this is for the first time that we are doing so, we better use libraries for handling sessions, since security is the primary aspect of any web application it is wiser to use existing (and good) packages to manage security.
 
 Path: `~/Tasks/sessions/sessions.go`
-
-    package sessions
-
-    import (
-        "net/http"
-
-        "github.com/gorilla/sessions"
-    )
-
-    //Store the cookie store which is going to store session data in the cookie
-    var Store = sessions.NewCookieStore([]byte("secret-password"))
-
-    //IsLoggedIn will check if the user has an active session and return True
-    func IsLoggedIn(r *http.Request) bool {
-        session, _ := Store.Get(r, "session")
-        if session.Values["loggedin"] == "true" {
-            return true
-        }
-        return false
-    }
-
+```golang
+package sessions
+
+import (
+"net/http"
+
+"github.com/gorilla/sessions"
+)
+
+//Store the cookie store which is going to store session data in the cookie
+var Store = sessions.NewCookieStore([]byte("secret-password"))
+
+//IsLoggedIn will check if the user has an active session and return True
+func IsLoggedIn(r *http.Request) bool {
+session, _ := Store.Get(r, "session")
+if session.Values["loggedin"] == "true" {
+    return true
+}
+return false
+}
+```
 This is the sessions package which we will use in our application. 
 
 We create a CookieStore which stores the sessions information under the "sessions" in the browser. We get the session ID stored under the session cookie and store it the `session` variable. When it comes to using this function, we have the `AddCommentFunc` view below which is going to handle the commenting feature of our application, it'll first check if the user has an active session and if so, it'll handle the POST request to add a comment, if not, it'll redirect the user to the login page.
 
 Path: `~/Tasks/Views/addViews.go`
+```golang
+//AddCommentFunc will be used
```

Comment:
I am confused because of this, I do not understand how that fencing works, if it looks good on Github, I feel we can skip the gofmt, because it would take too much effort to copy paste each block into a .go file and running gofmt on it! It would be very tedious. @WildYorkies I hope you just added ``` to the existing blocks without changing the entire code, that's the simplest way!